srcpkgs/protonvpn-cli/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'protonvpn-cli'
 pkgname=protonvpn-cli
 version=2.2.11
-revision=1
+revision=2
 wrksrc="linux-cli-community-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 makedepends="python3-devel"
-depends="openvpn python3-pythondialog python3-docopt python3-requests python3-Jinja2"
+depends="openvpn python3-pythondialog python3-docopt python3-requests python3-Jinja2 python3-distro"
```

Comment:
Please break this line so it's less than 80 columns.

Reply:
Done:  85bf358